src/assets/scss/components/_customInput.scss

```diff
@@ -1,4 +1,6 @@
 .custom-input {
+    display: inline-grid;
+    margin: 2.5px;
     input[type=checkbox]:checked ~ label {
         border: 2px $accent solid;
```

Comment:
Just realised if you create a faux border, the checkboxes won't flicker/move when they're clicked. You could do something like: `box-shadow: inset 0 0 0 -2px $accent`